stagemonitor-tracing/src/test/java/org/stagemonitor/tracing/AbstractRequestMonitorTest.java

```diff
@@ -95,13 +98,20 @@ public void before() {
 				return NoopTracerFactory.create();
 			}
 		});
-		assertThat(tracingPlugin.getTracer().scopeManager().active()).isNull();
+		assertThat(tracingPlugin.getTracer().scopeManager().activeSpan()).isNull();
 	}
 
 	@After
 	public void after() {
 		Stagemonitor.getMetric2Registry().removeMatching(Metric2Filter.ALL);
+		GlobalTracerTestHelper.resetGlobalTracer();
 		Stagemonitor.reset();
-		assertThat(tracingPlugin.getTracer().scopeManager().active()).isNull();
+		// TODO check if this is correct
+		boolean active = corePlugin.isStagemonitorActive();
+		if (active) {
+			assertThat(tracingPlugin.getTracer().scopeManager().activeSpan()).isNull();
```

Comment:
No active span when StageMonitor is active but active span when StageMonitor is not active?
How does that work?